<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Path of Building</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainsplitter">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>2000</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>500</height>
        </size>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabTree">
        <attribute name="title">
         <string>&amp;Tree</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabSkills">
        <attribute name="title">
         <string>&amp;Skills</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabItems">
        <attribute name="title">
         <string>&amp;Items</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>24</height>
    </size>
   </property>
   <widget class="QMenu" name="menuWidget">
    <property name="title">
     <string>&amp;Builds</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionLight"/>
    <addaction name="actionDark"/>
    <addaction name="actionStandard"/>
   </widget>
   <addaction name="menuWidget"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Start a new build</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save this build</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Path of Building</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionTree">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tree</string>
   </property>
   <property name="toolTip">
    <string>Switch Tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSkills">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skills</string>
   </property>
   <property name="toolTip">
    <string>Switch Tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionItems">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Items</string>
   </property>
   <property name="toolTip">
    <string>Switch Tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNotes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
   <property name="toolTip">
    <string>Switch Tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFive">
   <property name="text">
    <string>Five</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
   <property name="whatsThis">
    <string>light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
   <property name="whatsThis">
    <string>dark</string>
   </property>
  </action>
  <action name="actionFusion">
   <property name="text">
    <string>Fusion</string>
   </property>
   <property name="whatsThis">
    <string>Fusion</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="text">
    <string>Standard</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
